crates/nu-cli/tests/completions/mod.rs

```diff
@@ -329,6 +333,39 @@ fn file_completions() {
     // Match the results
     match_suggestions(&expected_paths, &suggestions);
 
+    // Test completions for the current folder even with parts before the autocomplet
+    let target_dir = format!("cp somefile.txt {dir_str}{MAIN_SEPARATOR}");
+    let suggestions = completer.complete(&target_dir, target_dir.len());
+
+    // Create the expected values
+    let expected_paths: Vec<String> = vec![
+        folder(dir.join("another")),
+        file(dir.join("custom_completion.nu")),
+        folder(dir.join("directory_completion")),
+        file(dir.join("nushell")),
+        folder(dir.join("test_a")),
+        folder(dir.join("test_b")),
+        file(dir.join(".hidden_file")),
+        folder(dir.join(".hidden_folder")),
+    ];
+
+    #[cfg(windows)]
+    {
+        let separator = '/';
+        let target_dir = format!("cp {dir_str}{separator}");
```

Comment:
This seems like the same test as above, did you mean to have `"cp somefile.txt {dir_str}{separator}"` instead?

Reply:
True.